core-bundle/src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -794,6 +800,8 @@ public function cutAll()
 		{
 			foreach ($arrClipboard[$this->strTable]['id'] as $id)
 			{
+				// No need for permission check here, it's done in cut()
+
```

Comment:
should try-catch and skip if a `cut` operation is denied

Reply:
Done in 596477a7322a89937cf775075b03c7521be9d120.